config/networking/networkpolicy.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: telemetry-operator-pprof-deny-ingress
```

Comment:
@hisarbalik the `telemetry-` prefix will be added by Kustomize https://github.com/kyma-project/telemetry-manager/blob/main/config/default/kustomization.yaml#L9. So, the name here should be only `operator-pprof-deny-ingress`